modules/declarative-configuration-auth-resources.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-role-based-access-control-3630.adoc
+:_content-type: CONCEPT
+[id="declarative-configuration-auth-resources_{context}"]
```

Comment:
Wondering where a good section would be for this doc, right now its under `Managing user access`, and while its currently only possible to create authN/Z resources, the scope will probably extend further in the future. Maybe it should be its own section, since things like setting it up, creating the config / linting, as well as troubleshooting will be generic irrespective of whether it's going to be an authN/Z resource or say a Policy?

Reply:
Reorganized and added as its own section under "Configuring" - let me know if there is a better place....